### PR TITLE
Action Metadata Example

### PR DESCRIPTION
**Pull request by Ryun1**
Hey Pi,

I started learning about json-ld and wanted to start making some mock data so we can start building tooling. So I created this mock action example.

Thought this may help😎.

### CODE REVIEW

**Comment by Ryun1** [comment]
Cross tagging https://github.com/cardano-foundation/CIPs/pull/556.

**Comment by Quantumplation** [comment]
Ooo, fantastic, thank you :)

I think there's a way to specify that you're importing *all* of the definitions from a file in the @context, rather than importing each one explicitly.

FWIW Sundae Labs has been offered a grant to build a Rust client library for dealing with this governance metadata; just wanted to make sure we aren't duplicating effort there!

**Comment by Ryun1** [comment]
Im glad to help.

> I think there's a way to specify that you're importing _all_ 

I had assumed so, but I couldn't figure it out, Im sure there is also other obvious errors.

> FWIW Sundae Labs has been offered a grant to build a Rust client library for dealing with this governance metadata; just wanted to make sure we aren't duplicating effort there!

Yup I am aware, looking forward to seeing that. I mainly started this so that I have something to show people when I talk about about a metadata anchors.

File: CIP-0100/action-example.jsonld [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "@context": {
+    "paymentAddress": "https://github.com/cardano-foundation/CIPs/blob/master/CIP-0100/common-properties.jsonld#payment-address",
+    "displayName": "https://github.com/cardano-foundation/CIPs/blob/master/CIP-0100/common-properties.jsonld#displayName",
+    "DID": "https://github.com/cardano-foundation/CIPs/blob/master/CIP-0100/common-properties.jsonld#DID",
+    "language": "https://datahub.io/core/language-codes/r/0.html",
```
Comment (Quantumplation) [inline CIP-0100/action-example.jsonld:6]: It looks like json-ld has native understanding of BCP47 language codes: https://www.w3.org/ns/json-ld#language

It also has support for providing multiple translations for *any* field:

https://www.w3.org/TR/json-ld11/#language-indexing

I may update the draft with these in mind.

**Pull request closed without merging**